components/org.wso2.carbon.appmgt.gateway/src/main/java/org/wso2/carbon/appmgt/gateway/token/AbstractJWTGenerator.java

```diff
@@ -142,7 +144,7 @@ public AbstractJWTGenerator() {
      * @param webApp
      * @param messageContext
      * @return jwt token
-     * @throws org.wso2.carbon.appmgt.api.AppManagementException
+     * @throws AppManagementException
```

Comment:
Formatting issues and Java doc comment issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files